go/tasks/plugins/hive/config/config.go

```diff
@@ -26,33 +26,42 @@ func MustParse(s string) config.URL {
 }
 
 type ClusterConfig struct {
-	Label string `json:"label" pflag:",Label of the service cluster"`
-	Limit int    `json:"limit" pflag:",Resource quota (in the number of outstanding requests) of the service cluster"`
+	PrimaryLabel string   `json:"primaryLabel" pflag:",The primary label of a given service cluster"`
+	Labels       []string `json:"labels" pflag:",Labels of a given service cluster"`
+	Limit        int      `json:"limit" pflag:",Resource quota (in the number of outstanding requests) of the service cluster"`
+}
+
+type ProjectDestinationClusterConfig struct {
```

Comment:
Can you rename to either ProjectDomainDestinationClusterConfig?  Or maybe just DestinationClusterConfig?